src/view/modals/TransferFactory.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  DialogTitle,
+  Dialog,
+  DialogContent,
+  DialogContentText,
+  IconButton,
+  Grid,
+} from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+import { useBatchStatus } from 'services/hooks';
+import { FactoryResData } from 'services/api';
+import { UpdateOrganizationForm } from 'view/forms/organization';
+import { DialogProps } from './utils';
+
+export interface TransferFactoryDialogProps extends DialogProps {
+  existingOrg: FactoryResData;
+}
+
+export const TransferFactoryDialog = ({
+  open,
+  handleClose,
+  existingOrg,
+}: TransferFactoryDialogProps) => {
+  const { setBatchStatusLink } = useBatchStatus();
+
+  return (
+    <Dialog
+      onClose={handleClose}
+      aria-labelledby="transfer-factory-title"
+      scroll="body"
+      fullWidth
+      maxWidth="md"
+      open={open}
+    >
+      <DialogTitle>
+        <Grid container justify="space-between">
+          <h2>Confirm Factory Information</h2>
```

Comment:
In general, we should use the typography component over vanilla HTML. To be honest, I don't have a great answer as to why that is the case, but I think it more or less boils down to the benefits of doing CSS in JS vs vanilla CSS.

E.g. if we want to change the style of all h2 elements, if we were using vanilla `<h2 />` tags, we could just set a global h2 style in CSS. Or, we could update our MUI theme object, and then all Typography components with `variant="h2"` would update. 

Here's a [link](https://medium.com/@_imGufran/why-use-css-in-js-benefits-and-drawbacks-df40ab4ebef9) if you want to dig into it more.

```suggestion
          <Typography variant="h2">Confirm Factory Information</Typography>
```